README.md

```diff
@@ -1,99 +1,65 @@
-# Polaris Collection for Ansible
-<!-- Add CI and code coverage badges here. Samples included below. -->
-[![CI](https://github.com/linkorb/ansible-collection-polaris/workflows/CI/badge.svg?event=push)](https://github.com/linkorb/ansible-collection-polaris/actions) [![Codecov](https://img.shields.io/codecov/c/github/linkorb/ansible-collection-polaris)](https://codecov.io/gh/linkorb/ansible-collection-polaris)
+<!-- Managed by https://github.com/linkorb/repo-ansible. Manual changes will be overwritten. -->
+ansible-collection-polaris
```

Comment:
I think we should keep the level 1 heading as it its.  Let's also keep a consistent markup (`#` instead of `===`).

Reply:
This is the format that was defined in repo-ansible, and what it autogenerates. We can create a task for further enhancement on how the README is generated, but for now should be merged as is.

Reply:
OK.